monGARS/core/iris.py

```
@@ -1,57 +1,337 @@
+    def _extract_document(self, response: httpx.Response) -> IrisDocument | None:
+        extracted_json: str | None = None
+        try:
+            extracted_json = trafilatura.extract(
+                response.text,
+                include_comments=False,
+                include_tables=False,
+                favor_precision=True,
+                output_format="json",
+                url=str(response.request.url),
+            )
+        except Exception as exc:  # pragma: no cover - library edge case
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)